.github/workflows/ci.yml

```diff
@@ -52,7 +52,12 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest, ubuntu-24.04-arm]
+        os: 
+          - macos-15-intel
+          - macos-latest
+          - windows-latest
+          - ubuntu-latest
+          - ubuntu-24.04-arm
```

Comment:
while I was at it I also expanded this one liner to improve how useful `git blame` is in the future.